prismic/custom-types/badger.json

```diff
@@ -60,16 +60,38 @@
         "label" : "Github"
       }
     },
+    "primary-image" : {
+      "type" : "Image",
+      "config" : {
+        "constraint" : {
+          "width" : 670,
+          "height" : 670
```

Comment:
It's going to be cropped but original image is still accessible in prismic to edit the crop